.github/workflows/Student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+    attr_reader :classroom
+  
+    def initialize(classroom:, name: "Unknown", age:, parent_permission: true)
+      super(name: name, age: age, parent_permission: parent_permission)
```

Comment:
- [ Optional ] Please I don't see the purpose of adding name: age:,parent_permission: symbols here I think ruby is smart enough to match all these attributes with the parent class